docs/code-contributions.md

```diff
@@ -18,6 +18,9 @@ If you manage packages with Homebrew you can do the following:
 brew install python3 python-setuptools
 ```
 
+> [!TIP]
+> If you encounter the error `Error: Cannot find module 'appdmg'`, ensure that `setuptools` is installed in your environment.
```

Comment:
```suggestion
> If you encounter `Error: Cannot find module 'appdmg'` error, ensure that `python-setuptools` are installed in your environment according to the previous steps.
```

I suggest to move it to the end of 'Running the App' section, too, as this error would occur at that time.

Reply:
Good point, adjusted and moved to the end of 'Running the App'